proxies/live/apiproxy/policies/AssignMessage.Errors.ServiceUnavailable.xml

```diff
@@ -21,10 +21,7 @@
   </AssignVariable>
   <AssignVariable>
     <Name>pds.error.diagnostics</Name>
-    <Value>
-      The downstream domain processing has not completed within the configured timeout period.
-      Retry your request after the time specified by the 'Retry-After' header, using the same 'X-Request-ID'.
-    </Value>
+    <Value>The downstream domain processing has not completed within the configured timeout period. Retry your request after the time specified by the 'Retry-After' header, using the same 'X-Request-ID'.</Value>
```

Comment:
is it worth mentioning that the _retry-after_ header is a response header?

`Retry your request after the time specified by the 'Retry-After' response header`

also maybe put the 'using' bit first i.e.  `Using the same 'X-Request-ID' header, retry your request after the time specified by the 'Retry-After' response header`

Reply:
Unless there's a formatting rule we follow that I don't know about then yeah we can change that.